while(true) fn foo() {}   // expect error
